RELEASE.md

```diff
@@ -4,15 +4,8 @@
 
 ### Features 🌈
 
-- New APIVersion: apps.emqx.io/v1beta2
-- Restart pods when update `configmap` and `secret`
-- Retain PVC when deleting EMQ X Custom Resource
-- The emqx operator controller automatically creates the `serviceAccount`, `role` and `roleBinding` resources when the resource specified by `.spec.serviceAccountName` does not exist
-- `.spec.serviceAccountName` is no longer required in EMQ X Custom Resource
-- `.spec.replicas` is no longer required in EMQ X Custom Resource
-- Add extra volumes mount possibility
-- Add readiness probe for Statefulset
+- Validate the format of the image tag by admission webhooks, the tag of the image must match '^[0-9]+.[0-9]+.[0-9]+$'
 
 ### Fixes 🛠
 
-- Fix annotations for EMQ X Custom Resource not working
+- Fix ".spec.env" not working in v1beta2 API version
```

Comment:
Maybe the Issue[How to add custom ENV in CRD EmqxBroker] (https://github.com/emqx/emqx-operator/issues?q=is%3Aissue+is%3Aclosed) attached better?